docs/governance/politeia.md

```diff
@@ -2,54 +2,55 @@
+6. When the ticket-voting period ends, the proposal is formally approved or rejected. There is a quorum requirement for a vote to be considered valid: 20% of the eligible tickets must vote 'Yes' or 'No'. The threshold for a proposal to be approved is 60% 'Yes' votes.
+7. When a proposal with a budget and deliverables is approved, work can begin. The proposal owner can submit claims against the budget as deliverables are completed.
+8. Payment claims will be handled manually by Decred Holdings Group (DHG) until such times as this process can be handled by the [Decred Decentralized Autonomous Entity (DAE)](#decentralized-autonomous-entity-dae).
 
-For now, the process is something like this:
+For guidance on creating a good proposal, see the [proposal guidelines](../governance/politeia/proposal-guidelines.md) and [example proposals](../governance/politeia/example-proposals.md) pages.
 
-1. Submit proposal.
-2. Proposal reviewed by Politeia admins, spam is censored.
-3. Valid proposals appear publicly on Politeia, open for discussion, but voting does not begin immediately.  	
-4. Ticket-voting interval (~1 week) can be triggered by the proposal owner. Until ticket-voting is triggered, the proposal can be edited by its owner in response to community feedback.
-5. Ticket-voting starts, ticket-holders vote through their wallet. Every ticket that was live 256 blocks before voting started is eligible to vote on the proposal. Votes by eligible tickets can be cast any time within the voting interval. Once votes are cast they cannot be changed.
-6. When the ticket-voting period ends, the proposal is formally approved or rejected. There is a quorum requirement of 20% tickets voting Yes/No. For a proposal to pass it must achieve 60% Yes votes.
-7. When a proposal with a budget and deliverables is approved, work can begin. The proposal owner can submit claims against that budget as deliverables are completed.
-8. Payment claims will be handled manually by Decred Holdings Group until such times as this process can be adequately decentralized.
+### Decentralized Autonomous Entity (DAE)
+
+Politeia's aim is to serve as the decision-making force behind the Decred Decentralized Autonomous Entity (DAE). This is an ambitious aim. Because Politeia and its accompanying processes are still in an experimental stage, Politeia will serve initially as a robust signaling mechanism from Decred shareholders to the community members controlling the Decred Treasury wallet. Over time, this will be iterated towards a DAE that decentralizes the functions currently performed by trusted humans, and the DAE will control the project fund wallet directly.
```

Comment:
> This is an ambitious aim.

This feels a bit subjective for a site which is mostly technical documentation. I would leave it out personally.

Reply:
Removed.

Reply:
I noticed that you deleted this whole section. I was only referring to the bit `"This is an ambitious aim."`

I think the rest of the paragraph was good. Its up to you if you want to remove it or leave it in.